core/src/main/java/com/linecorp/armeria/internal/common/Http2ObjectEncoder.java

```diff
@@ -120,6 +124,21 @@ protected final boolean isStreamPresentAndWritable(int streamId) {
         }
     }
 
+    protected boolean isEndStreamSent(Http2Stream stream) {
+        return stream.getProperty(endStreamSentKey) != null;
+    }
+
+    protected void maybeSetEndStreamSent(int streamId, boolean endStream) {
+        if (!endStream) {
+            return;
+        }
+        final Http2Stream stream = encoder.connection().stream(streamId);
+        if (stream == null) {
+            return;
+        }
+        stream.setProperty(endStreamSentKey, true);
```

Comment:
Is this set only once for a connection?
For example, if `endStreamSentKey` is set to `true` for `streamId` `3`, then the following streams (e.g. `5`, `7`.. ) are all set to `true`?